packages/@aws-cdk/aws-cloud9-alpha/README.md

```diff
@@ -105,7 +105,16 @@ Every Cloud9 Environment has an **owner**. An owner has full control over the en
 
 By default, the owner will be the identity that creates the Environment, which is most likely your CloudFormation Execution Role when the Environment is created using CloudFormation. Provider a value for the `owner` property to assign a different owner, either a specific IAM User or the AWS Account Root User.
 
-`Owner` is a user that owns a Cloud9 environment . `Owner` has their own access permissions, resources. And we can specify an `Owner`in an Ec2 environment which could be of two types, 1. AccountRoot and 2. Iam User. It allows AWS to determine who has permissions to manage the environment, either an IAM user or the account root user (but using the account root user is not recommended, see [environment sharing best practices](https://docs.aws.amazon.com/cloud9/latest/user-guide/share-environment.html#share-environment-best-practices)).
+`Owner` is an IAM entity that owns a Cloud9 environment . `Owner` has their own access permissions, resources. You can specify an `Owner`in an Ec2 environment which could be of the following types:
+
+1. Account Root
+2. IAM User
+3. IAM Federated User
+4. IAM Assumed Role
+
+The ARN of the owner must satisfy the following regular expression: `^arn:(aws|aws-cn|aws-us-gov|aws-iso|aws-iso-b):(iam|sts)::\d+:(root|(user\/[\w+=/:,.@-]{1,64}|federated-user\/[\w+=/:,.@-]{2,32}|assumed-role\/[\w+=:,.@-]{1,64}\/[\w+=,.@-]{1,64}))$`
```

Comment:
I think this should be part of docs for the property itself and not the Readme

Reply:
I considered this, but I feel that this makes the docs very verbose while adding little value for the user

By adding the two functions `assumedRole(..)` and `federatedUser(..)` I abstract that very complexity away from the users. They now only need to know about the account id and the user / role name they want to use. For both the Regex adds no value since they need to refer to existing users / roles that are inherently well-formatted

I even considered leaving the regex out entirely, but ultimately I think it's worth to mention it once